app/helpers/search_helper.rb

```diff
@@ -64,6 +64,22 @@ def display_last_modified(record)
     end
   end
 
+  def display_tag_count(record)
+    tag_keys = record.fetch('added_fields', {}).fetch('tags', {})&.keys
+    tag_count = tag_keys.nil? ? 0 : tag_keys.count
```

Comment:
Since `tag_keys` will be `nil` if `&.keys` is not valid and the optional return parameter `{}` passed to `fetch` will not be used I think it would be cleaner to use `#dig`. This could be used `tag_keys = record.dig('added_fields', 'tags')&.keys` since `#dig` returns `nil` when one of the passed keys is not found